templates/service-location-page-2/service-location-page-2.js

```diff
@@ -38,7 +38,7 @@ function addServiceCards(main) {
     const button = document.createElement('a');
     button.textContent = 'Get a Quote';
     button.ariaLabel = button.textContent;
-    button.href = '#'; // TODO: add action to open the get-a-quote form
+    button.href = '/forms/modals/modal';
```

Comment:
@dkuntze - Is there a specific reason why the content for this service card is being pulled from the code instead of being authored through a document?